Stuart Hughes stuarth@freescale.com 18mar05

This fixes: 

gcc -O2 -DFACILITY=LOG_DAEMON    -DHOSTS_ACCESS  -DNETGROUP    -DDAEMON_UMASK=022 -DREAL_DAEMON_DIR=\"/usr/sbin\" -DPROCESS_OPTIONS -DKILL_IP_OPTIONS -DSEVERITY=LOG_INFO        -DRFC931_TIMEOUT=10  -DHOSTS_DENY=\"/etc/hosts.deny\" -DHOSTS_ALLOW=\"/etc/hosts.allow\"   -DSYS_ERRLIST_DEFINED -DHAVE_WEAKSYMS -D_REENTRANT    -o scaffold.o -c scaffold.c
scaffold.c:28: error: conflicting types for 'malloc'
scaffold.c:28: error: conflicting types for 'malloc'
make[1]: *** [scaffold.o] Error 1

This is seen on gcc-3.4 compilers


--- tcp_wrappers_7.6/scaffold.c	2005-03-18 08:02:08.000000000 -0600
+++ tcp_wrappers_7.6.modified/scaffold.c	2005-03-18 08:01:17.000000000 -0600
@@ -25,7 +25,8 @@
 #define	INADDR_NONE	(-1)		/* XXX should be 0xffffffff */
 #endif
 
-extern char *malloc();
+#include <stdlib.h>
+//extern char *malloc();
 
 /* Application-specific. */
 
